### ISSUE TITLE
Issue with CUDA Provider

### ISSUE THREAD

**Issue by Aestheral1** [C0untFloyd/roop-unleashed#361]
**Describe the bug**
I've already installed CUDA + cuDNN and started using it with 'python run.py --execution-provider cuda,' however, the GPU usage is quite low, and the CPU usage is very high, which doesn't make sense. I've tried a clean installation, but there have been no changes, is there any way to fix this and make it use only the GPU?

**Details**
What OS are you using?
- [ ] Linux
- [ ] Linux in WSL
- [x] Windows 
- [ ] Mac

Are you using a GPU?
- [ ] No. CPU FTW
- [x] NVIDIA RTX 2060 12GB
- [ ] AMD
- [ ] Intel
- [ ] Mac

**Which version of roop unleashed are you using?** 3.3.4

**Screenshots**

![cpu_gpu_usage](https://github.com/C0untFloyd/roop-unleashed/assets/127466775/d5f8b50a-5815-4915-a0d8-33e1a92a5fba)
![settings](https://github.com/C0untFloyd/roop-unleashed/assets/127466775/42998b5d-e0d4-4868-bd04-999a61dea737)

**Comment by lysxelapsed** [comment]
You can leave out "--execution-provider cuda", CLI arguments aren't supported anymore. Besides that:
- Are you using an enhancer and which one? 4 Threads are too much for 12GB VRAM with most enhancers.
- I wrote an extended piece on GPU utilization in the [FAQ ](https://github.com/C0untFloyd/roop-unleashed/wiki/FAQ) 2 days ago.

**Comment by Aestheral1** [comment]
Thank you for the CLI warning. I found the issue. I had recently formatted Windows, and before that, the GPU acceleration option in Windows was enabled, so it used the GPU for processing normally. However, after the format, I forgot to re-enable this option in Windows. Even with CUDA and cuDNN properly installed, the GPU usage percentage was low, but VRAM consumption was high because this option was not activated. I didn't find this mentioned in the Wiki, so I think it's important to leave a reminder so that it doesn't happen to someone unaware like me 😅
![image](https://github.com/C0untFloyd/roop-unleashed/assets/127466775/6d83edcc-02db-4f5e-97f9-efa76714c6c2)
